current/advanced/block-import.md

```diff
@@ -0,0 +1,86 @@
+---
+slug: block-import
+lang: en
+title: The Block Import Pipeline
+---
+
+## The Import Queue
+
+The import queue is an abstract worker queue present in every Substrate node. It is not part of the
+runtime. The Import Queue is responsible for processing pieces of incoming information, verifying
```

Comment:
```suggestion
runtime. The import queue is responsible for processing pieces of incoming information, verifying
```

Reply:
Dang, I was sure I got all of these. Good catch.